Updated ddt data from 2 to 3

adnanghaffar07 opened the pull request:
I have changed the ddt data form 2 to 3 and it is working on chrome and FF

adnanghaffar07 commented:
@benpatterson @jzoldak @raeeschachar Please review this, 

I have changed the ddt data and it is working fine on chrome.

benpatterson commented:
how was this failing on chrome? Could you explain how this fixes it?

adnanghaffar07 commented:
I have replaced the data 1 with 2, then 2 to 3 so on. e.g     

before updating: @ddt.data(1, 2, 3)
after updating:  @ddt.data(2, 3, 4)

benpatterson commented:
Hmm the change appears to be to 2, 3, 3 instead. 

But still I don't understand _how_ this fixes it. Cloud you explain?

benpatterson commented:
s/cloud/could

adnanghaffar07 commented:
Well, this test is related to below mentioned test.
lms/test_library.py:LibraryContentTest.test_shows_random_xblocks_from_configured_1_2	
 
In first test I changed the test data 1 to 2 and other changes are in count function e.g to select all then clear. When we remove value and enter same value, Chrome browser don't gets its changes. 

Count function have been calling in all these three tests and this test was fixed automatically, because I have amended the count function. 

After change the test data 1 to 2, its shows  @ddt.data(2, 2, 3), now it dons't make sense that we are running the test with same data, then I changed the all data in ddt " @ddt.data(2, 3, 4)".

About this test: I did make the same changes for this test because it was appending the value instead of replacement and then changed the data.

Now all three tests are working good on chrome and firefox.

1- lms/test_library.py:LibraryContentTest.test_shows_random_xblocks_from_configured_1_2	
2- lms/test_library.py:LibraryContentTest.test_shows_random_xblocks_from_configured_2_3	
3- lms/test_library.py:LibraryContentTest.test_shows_random_xblocks_from_configured_3_4

adnanghaffar07 commented:
I am making the branches from one to each other that's why it is showing 2, 3, 3 in this PR but when I will merged them all it would be 2,3,4
